src/main/java/teammates/ui/controller/StudentFeedbackResultsPageAction.java

```diff
@@ -47,9 +49,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         }
 
         if (data.getBundle().isStudentHasSomethingNewToSee(data.student)) {
-            statusToUser.add(Const.StatusMessages.FEEDBACK_RESULTS_SOMETHINGNEW);
+            statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_RESULTS_SOMETHINGNEW, StatusMessageColor.INFO));
         } else {
-            statusToUser.add(Const.StatusMessages.FEEDBACK_RESULTS_NOTHINGNEW);
+            statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_RESULTS_NOTHINGNEW, StatusMessageColor.WARNING));
```

Comment:
The message is "You have not received any new feedback but you may review your own submissions below." I used WARNING to warn the student that there is no feedback.